src/util/common.jl

```diff
@@ -0,0 +1,308 @@
+
+"updates the solve time limit of an optimizer"
+function _update_optimizer_time_limit!(optimizer::MathOptInterface.OptimizerWithAttributes, time_limit::Real)
+        new_params = typeof((optimizer.params))()
+    for param in optimizer.params
+        if param[1] != MathOptInterface.TimeLimitSec()
+            push!(new_params, param)
+        end
+    end
+    empty!(optimizer.params)
+    for param in new_params
+        push!(optimizer.params, param)
+    end
+    push!(optimizer.params, (MathOptInterface.TimeLimitSec()=>time_limit))
+
```

Comment:
Can we use the JuMP interface instead? https://jump.dev/JuMP.jl/stable/manual/models/#Set-a-time-limit

Maybe we don't need this helper function if the JuMP interface suits our needs?

Reply:
The JuMP interface works for this use.  MOI requirement is dropped.